app/views/exchange_rates/show_404.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :top_breadcrumbs do %>
+   <%= generate_breadcrumbs "Exchange rates #{@monthly_exchange_rate.month_and_year_name}",
+                            [ [t('breadcrumb.home'), home_path],
+                            [t('breadcrumb.tools'), tools_path],
+                            [t('breadcrumb.exchange_rates'), exchange_rates_path], ] %>
+ <% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <p>
+      There are no exchnage rates present for the year requested please use the back navigation for years with data.
```

Comment:
```suggestion
      There are no exchange rates present for the year requested please use the back navigation for years with data.
```

Reply:
Actually this has been pushed with old content, but yes i still make the mistake when typing `exchange` 😶‍🌫️